### PR TITLE
Add SharedKey parameter in Add/New/Set BGP commands 

### PR DESCRIPTION
**Pull request by dihan0604**
Provider the optional parameter sharedkey to customer when they add/new/set bgp command

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@dihan0604__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@dihan0604__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@dihan0604 Thanks for your contribution, we need to have the service team review this PR before we can accept it

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Peering/AzureExpressRouteCircuitPeeringConfigBase.cs [hunk 1]
```diff
@@ -64,6 +64,12 @@ public class AzureExpressRouteCircuitPeeringConfigBase : NetworkBaseCmdlet
 
         [Parameter(
             Mandatory = false,
+            HelpMessage = "The shareKey")]
```
Comment (nupurkot) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Peering/AzureExpressRouteCircuitPeeringConfigBase.cs]: Minor typo: The shared key

**Comment by nupurkot** [comment]
LGTM once the minor typo in help message has been fixed

**Comment by markcowl** [comment]
@dihan0604 Please respond to feedback

**Comment by nupurkot** [comment]
LGTM

**Comment by dihan0604** [comment]
I have fixed the typo and Nupur has signed off.

**Comment by hovsepm** [comment]
on-demand job - http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/284/

**Comment by markcowl** [comment]
updated on demand job:  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/286/

**Comment by markcowl** [comment]
@dihan0604 Please retarget your PR at the release branch release-1.0.0 to include it in this release.

**Comment by dihan0604** [comment]
Do you mean release-1.0.0-preview2?  This is the only branch I see for release

**Pull request closed without merging**